index.html

```diff
@@ -207,17 +207,20 @@ <h4>Skills</h4>
                         </ul>
                     </div>
                 </li>
-            </ul>
-            
+            </ul>        
+        </section>
+
+        <section id="contact-form">
```

Comment:
- [ ] Kindly make sure your design matches the Figma, therefore I suggest adding the background image that is present in the right of this section :recycle:  
![Screenshot from 2023-01-18 00-35-38](https://user-images.githubusercontent.com/97834160/213027620-6cc7a898-7e7a-41f5-9017-82fcd39bba15.png)